src/gui/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ add_executable(zappy_gui
         cli/ClientApi.cpp
         cli/ParserData.hpp
         cli/ParserData.cpp
-        cli/ServerData.cpp
```

Comment:
you remove `ServerData.cpp` ?

Reply:
yeah, implementation are directly in hpp